openbb_terminal/stocks/technical_analysis/ta_controller.py

```diff
@@ -129,46 +129,48 @@ def print_help(self):
             stock_str = f"{s_intraday} {self.ticker}"
 
         mt = MenuText("stocks/ta/", 90)
-        mt.add_param("_ticker", stock_str)
+        mt.add_param(
```

Comment:
should we also have the `load` cmd here? the functionality is there tho

Reply:
yes i should